test/Azure.Functions.Cli.Tests/ActionsTests/ResolveActionTests.cs

```diff
@@ -52,6 +53,15 @@ public class ResolveActionTests
         [InlineData("azure functionapp --help", typeof(HelpAction))]
         [InlineData("azure --help", typeof(HelpAction))]
         [InlineData("--help", typeof(HelpAction))]
+        [InlineData("durable delete-task-hub --connection-string connection", typeof(DurableDeleteTaskHub))]
+        [InlineData("durable get-history --id 1234 --show-input true", typeof(DurableGetHistory))]
```

Comment:
Instead of `1234` as the example, I suggest using a real GUID.

Reply:
It's perfectly fine to use a hardcoded one. I just want the reader to recognize that most often these will be GUID values.